src/Helper/Placeholder/Container/AbstractContainer.php

```diff
@@ -118,6 +118,10 @@ public function toString($indent = null)
             : $this->getIndent();
 
         $items  = $this->getArrayCopy();
+        // If we don't have items - do not show prefix and postfix
```

Comment:
If no item is present, then this check should the first step in this method. You can use `$this->count()`.